src/main/java/uk/gov/hmcts/cmc/claimstore/documents/CitizenServiceDocumentsService.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.config.properties.pdf.DocumentTemplates;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.DefendantPinLetterContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.reform.sendletter.api.Document;
+
+@Service
+public class CitizenServiceDocumentsService {
```

Comment:
![meme](https://i.imgflip.com/25uwoe.jpg)

Reply:
Also it wound be nice to see a unit test for it.

Reply:
what would it do? I added the config to make the unit tests run this code

Reply:
I would just verify that created documents contain expected data.

Reply:
Also I was serious with that meme -- why is `Service` two times in that class name?

Reply:
two different meanings of `Service` couldn't think of a better name, unless you can?

Reply:
I would just drop first `Service`, should be fine in my opinion.